src/main/java/io/vlingo/actors/plugin/mailbox/telemetry/DefaultMailboxTelemetry.java

```diff
@@ -12,6 +12,8 @@
 import static java.util.Collections.singleton;
 
 public class DefaultMailboxTelemetry implements MailboxTelemetry {
+  private static final String LAG = "lag";
```

Comment:
Change "lag" to "pending" or if "lag" means "fell behind" the name it "behindBy".

Reply:
It's pending, forgot to change the name!